testcases/OpTestLPM.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: op-test-framework/testcases/OpTestNVRAM.py $
+#
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2015
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+#
+
+'''
+OpTestLPM
+---------
+
+This test is to preform or validate basic LPM migration
+from source to destination managed system 
+'''
+
+import unittest
+import time
+import OpTestConfiguration
+from common import OpTestHMC
+from common.OpTestSystem import OpSystemState
+
+
+class OpTestLPM(unittest.TestCase):
+
+    def setUp(self):
+        conf = OpTestConfiguration.conf
+        self.cv_SYSTEM = conf.system()
+        self.console = self.cv_SYSTEM.console
+        self.cv_HMC = self.cv_SYSTEM.hmc
+        self.src_mg_sys = self.cv_HMC.mg_system
+        self.det_mg_sys = self.cv_HMC.sec_mg_system
+
+    def lpm_migrate_test(self):
+        if self.cv_HMC.migrate_lpar(self.src_mg_sys, self.det_mg_sys):
+            log.info("Simple migration was successfull!!!")
+            return
+        raise OpTestError("Simple migration test failed")
+
+    def lpm_migrate_back_test(self):
+        if self.cv_HMC.migrate_lpar(self.det_mg_sys,  self.src_mg_sys):
+            log.info("Migration back was successfull")
+            return
+        raise OpTestError("Migration back test failed")
+
+    def runTest(self):
+        self.lpm_migrate_test()
+        self.lpm_migrate_back_test()
```

Comment:
It is highly recommended to have atleast 5 minutes gap between LPMs

Reply:
Agreed, Added 5 mins delay between migration.